src/lib.rs

```diff
@@ -612,6 +716,32 @@ impl Uuid {
         Urn { inner: self }
     }
 
+
+    /// Returns an Optional Tuple of (u64, u16) representing the timestamp and 
+    /// counter portion of a V1 UUID.  If the supplied UUID is not V1, this 
+    /// will return None
+    #[cfg(feature = "v1")]
```

Comment:
I think we don't have to gate this behind the v1 feature, right? If you create a raw uuid from bytes you may still wish to parse it out even though you're not creating uuids?